instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -98,6 +101,7 @@ def wrap_connect(
     connection_attributes: typing.Dict = None,
     version: str = "",
     tracer_provider: typing.Optional[TracerProvider] = None,
+    capture_parameters: bool = False,
```

Comment:
nitpick: with boolean values, I would suggest an auxiliary verb to indicate it's a boolean decision. 

e.g. "should_capture_parameters" .

Reply:
AsyncPGInstrumentor already introduce the parameter as capture_parameters.  I am trying to be consistent with that.